osu.Game/Screens/SelectV2/Footer/BeatmapOptionsPopover.cs

```diff
@@ -60,15 +60,15 @@ private void load(ManageCollectionsDialog? manageCollectionsDialog, SongSelect?
             addButton(SongSelectStrings.ManageCollections, FontAwesome.Solid.Book, () => manageCollectionsDialog?.Show());
 
             addHeader(SongSelectStrings.ForAllDifficulties, beatmapWhenOpening.BeatmapSetInfo.ToString());
-            addButton(SongSelectStrings.DeleteBeatmap, FontAwesome.Solid.Trash, () => songSelect?.DeleteBeatmap(beatmapWhenOpening.BeatmapSetInfo), colours.Red1);
+            // addButton(SongSelectStrings.DeleteBeatmap, FontAwesome.Solid.Trash, () => songSelect?.DeleteBeatmap(beatmapWhenOpening.BeatmapSetInfo), colours.Red1);
```

Comment:
What's the reason for disabling these?

Reply:
Just detaching all new components from old `SongSelect`, I'll bring back the buttons but remove the actions (functionality comes later).